shows_app/src/app/components/shared/SideBar/SidebarNavigation.tsx

```diff
@@ -1,57 +1,141 @@
 "use client";
 
+import React from "react";
 import {
   Box,
   Button,
-  Divider,
   Flex,
+  Img,
   Spacer,
   Stack,
-  Text
+  Text,
+  useBreakpointValue,
+  useDisclosure,
+  Drawer,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerHeader,
+  DrawerBody,
+  IconButton
 } from "@chakra-ui/react";
 import NextLink from "next/link";
 import { useRouter } from "next/navigation";
+import { CloseIcon, HamburgerIcon } from "@chakra-ui/icons";
 
 export const SideBarNavigation = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
   const router = useRouter();
 
   const handleLogout = () => {
     localStorage.removeItem("authToken");
     router.push("/login");
   };
 
+  const sidebarWidth = useBreakpointValue({
+    base: "100%",
+    md: "300px"
+  });
+
   return (
-    <Box
-      maxWidth="300px"
-      position="fixed"
-      left="0"
-      top="0"
-      bottom="0"
-      backgroundColor="brand.100"
-      color="white"
-      padding="4"
-      display="flex"
-      flexDirection="column"
-    >
-      <Stack spacing="4" flexGrow={1}>
-        <NextLink href="/all-shows">
-          <Text fontSize="lg">All shows</Text>
-        </NextLink>
-        <NextLink href="/all-shows/top-rated">
-          <Text fontSize="lg">Top rated</Text>
-        </NextLink>
-        <NextLink href="/profile">
-          <Text fontSize="lg">My profile</Text>
-        </NextLink>
-        <Divider />
-        <Spacer />
-      </Stack>
+    <>
+      <IconButton
+        icon={<HamburgerIcon />}
+        onClick={onOpen}
+        display={{ base: "block", md: "none" }}
```

Comment:
Instead of hiding this component, lets:
1. Make a SidebarNavigation component
2. Make a MobileNavigation component
3. Use the ChakraUI [Show](https://v2.chakra-ui.com/docs/components/show-hide/usage) component to show or hide the SidebarNavigation and MobileNavigation components on different breakpoints. `<Show below="md"><MobileNavigation...`